nmosregistration/common/routes.py

```diff
@@ -24,16 +24,19 @@
 from nmosregistration.modifier import RegModifier
 from nmosregistration.common import schema
 
+from requests.models import Response
+
 VALID_TYPES = ['node', 'source', 'flow', 'device', "receiver", "sender"]
 REGISTRY_PORT = 4001
 NODE_SEEN_TTL = 12  # seconds until a node considered "dead".
```

Comment:
Obviously fine to use these as they're already here, but 'REGISTRY_PORT' and 'NODE_SEEN_TTL' (which should really be 'heartbeat_timeout' or similar) would be better derived from the config.